webrtc_live_view/lib/webrtc_live_view/contours_drawer.ex

```diff
@@ -0,0 +1,45 @@
+defmodule WebRTCLiveView.CountoursDrawer do
+  use Membrane.Filter
+
+  alias Evision.{ColorConversionCodes, Constant}
+
+  def_input_pad :input, accepted_format: Membrane.RawVideo
+  def_output_pad :output, accepted_format: Membrane.RawVideo
+
+  @impl true
+  def handle_buffer(:input, buffer, ctx, state) do
+    %{height: height, width: width} = ctx.pads.input.stream_format
+
+    {:ok, image} =
+      Vix.Vips.Image.new_from_binary(buffer.payload, width, height, 3, :VIPS_FORMAT_UCHAR)
+
+    {:ok, image} = Image.to_evision(image)
+
+    grayscale = Evision.cvtColor(image, ColorConversionCodes.cv_COLOR_BGR2GRAY())
+    flags = Bitwise.bor(Constant.cv_THRESH_BINARY(), Constant.cv_THRESH_OTSU())
+    {_ok, bw} = Evision.threshold(grayscale, 50, 255, flags)
+
+    {contours, _} =
+      Evision.findContours(
+        bw,
+        Constant.cv_RETR_LIST(),
+        Constant.cv_CHAIN_APPROX_NONE()
+      )
+
+    contours =
+      Enum.filter(contours, fn c ->
+        trunc(Evision.contourArea(c)) in 100..200_000
```

Comment:
What is that purpose of that filtering?

Reply:
IDK, I took this code from @mat-hek

Reply:
These are some magic numbers but I don't think there is any value in assigning them names since we don't know if they have any semantic sense, so perhaps let's leave it as it is :D